presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemoryDistributedQueries.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.memory;
+
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+public class TestMemoryDistributedQueries
```

Comment:
Is not this an overkill? Maybe you could just copy couple of view tests into `TestMemorySmoke`, would not be that enough?

It is is going to make travis slower and could make some travis job to exceed time limit.

Things like that causes https://github.com/prestodb/presto/pull/8844 to be more and more important.